ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.18-nightly-x86.yaml

```diff
@@ -201,6 +201,44 @@ tests:
     test:
     - chain: openshift-qe-node-density-cni
     workflow: openshift-qe-installer-aws
+- as: control-plane-ipsec-120nodes
+  cron: 0 3 22 * *
+  steps:
+    allow_skip_on_success: true
+    cluster_profile: aws-perfscale
+    env:
+      BASE_DOMAIN: perfscale.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "120"
+      COMPUTE_NODE_TYPE: m5.xlarge
+      EXTRA_FLAGS: --churn-duration=20m --service-latency
+      PROFILE_TYPE: reporting
+      SIZE_VARIANT: large
+      USER_TAGS: |
+        TicketId 382
+      ZONES_COUNT: "3"
```

Comment:
Please add the following envs to specify a 16vCPU infra node size for 120 worker nodes cluster. 
```
SET_ENV_BY_PLATFORM: custom
OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: Standard_E16s_v3
```
Refer to this doc
https://docs.google.com/document/d/1sVkbU5ttS3g_GqfEeAWDzT25asvy_jKqy0qGCJ4Lfzo/edit#heading=h.x01a3v2u2rct

Reply:
120, 252 nodes based on the above linked document, refer to instance size of
```
OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: r5.4xlarge
```